docs/cluster-wide.md

```diff
@@ -0,0 +1,160 @@
+
+By default, Percona Operator for MongoDB functions in a specific Kubernetes
+namespace. You can create one during installation (like it is shown in the
+[installation instructions](kubernetes.md#install-kubernetes)) or just use the
+`default` namespace. This approach allows several Operators to co-exist in one
+Kubernetes-based environment, being separated in different namespaces:
+
+![image](assets/images/cluster-wide-1.svg)
+
+Still, sometimes it is more convenient to have one Operator watching for
+Percona Server for MongoDB Custom Resources in several namespaces.
+
+We recommend running Percona Operator for MongoDB in a traditional way,
+limited to a specific namespace. But it is possible to run it in so-called
+*cluster-wide* mode, one Operator watching several namespaces, if needed:
+
+![image](assets/images/cluster-wide-2.svg)
+
+!!! note
+
+    Please take into account that if several Operators are configured to
+    watch the same namespace, it is entirely unpredictable which one will get
+    ownership of the Custom Resource in it, so this situation should be avoided.
+
+To use the Operator in such *cluster-wide* mode, you should install it with a
+different set of configuration YAML files, which are available in the `deploy`
+folder and have filenames with a special `cw-` prefix: e.g.
+`deploy/cw-bundle.yaml`.
+
+While using this cluster-wide versions of configuration files, you should set
+the following information there:
+
+* `subjects.namespace` option should contain the namespace which will host
+    the Operator,
+* `WATCH_NAMESPACE` key-value pair in the `env` section should have
+    `value` equal to a  comma-separated list of the namespaces to be watched by
+    the Operator, *and* the namespace in which the Operator resides (or just a
+    blank string to make the Operator deal with *all namespaces* in a Kubernetes
+    cluster).
+
+The following simple example shows how to install Operator cluster-wide on
+Kubernetes.
+
+1. First of all, clone the percona-server-mongodb-operator repository:
+
+    ```bash
+    $ git clone -b v{{ release }} https://github.com/percona/percona-server-mongodb-operator
+    $ cd percona-server-mongodb-operator
+    ```
+
+2. Let’s suppose that Operator’s namespace should be the `psmdb-operator` one.
+    Create it as follows:
+
+    ```bash
+    $ kubectl create namespace psmdb-operator
+    ```
+
+    Namespaces to be watched by the Operator should be created in the same way
+    if not exist. Let’s say the Operator should watch the `psmdb` namespace:
+
+    ```bash
+    $ kubectl create namespace psmdb
+    ```
+
+3. Edit the ``deploy/cw-bundle.yaml`` configuration file to set proper
+    namespaces:
+
+    ```yaml
+    ...
+    subjects:
+    - kind: ServiceAccount
+      name: percona-server-mongodb-operator
+      namespace: "psmdb-operator"
+    ...
+    env:
+             - name: WATCH_NAMESPACE
+               value: "psmdb"
+    ...
+    ```
```

Comment:
Well we can start with simpliest intallation without using this var. And then explain that we can use it if we want to limit the namespaces operator can see.
We need to list in this var operator ns and watched nss (as for PXC)

Reply:
It was already explained in the lines 36-40. Here we just put some example to look at